UI/platform-windows.cpp

```diff
@@ -524,19 +524,22 @@ void TaskbarOverlaySetStatus(TaskbarOverlayStatus status)
 		return;
 	}
 
-	if (status != TaskbarOverlayStatusInactive) {
-		QIcon qicon;
-		switch (status) {
-		case TaskbarOverlayStatusActive:
-			qicon = QIcon::fromTheme(
-				"obs-active", QIcon(":/res/images/active.png"));
-			break;
-		case TaskbarOverlayStatusPaused:
-			qicon = QIcon::fromTheme(
-				"obs-paused", QIcon(":/res/images/paused.png"));
-			break;
-		}
+	QIcon qicon;
+	switch (status) {
+	case TaskbarOverlayStatusActive:
+		qicon = QIcon::fromTheme("obs-active",
+					 QIcon(":/res/images/active.png"));
+		break;
+	case TaskbarOverlayStatusPaused:
+		qicon = QIcon::fromTheme("obs-paused",
+					 QIcon(":/res/images/paused.png"));
+		break;
+	case TaskbarOverlayStatusInactive:
+		taskbarIcon->SetOverlayIcon(hwnd, nullptr, nullptr);
+		break;
```

Comment:
How about return here so that it's not necessary to check `status` again later.
```suggestion
		taskbarIcon->Release();
		return;
```

Reply:
I finish my work about MSVC warning and then I will apply your suggestion.